Multiple repository url (#1483)
kyma-project/kyma: docs/service-brokers/docs/011-configuration-helm-broker.md

    - A `{bundle_name}-{bundle_version}.tgz` file for each bundle version defined in the `yaml` file. The `.tgz` file is an archive of your bundle's directory.

> 2. In the `values.yaml` provide your server's URLs in the **repository.URLs** attribute as a list of URLs separated by a semi-colon:

Reviewer: ```suggestion
2. In the `values.yaml` provide your server's URLs in the **repository.URLs** attribute as a list of URLs separated by a semicolon:
```

  ```yaml
    repository:
      URLs: "http://custom.bundles-repository/bundles.yaml;http://another.bundles-repository/bundles.yaml"
  ```
  > **NOTE:** You can skip the `yaml` filename in the URL if the name of the file is `index.yaml`. In that case, your URL should be equal to `http://custom.bundles-repository/`.

3. Install Kyma on Minikube. See the **Install Kyma locally from the release** document to learn how.

### Configure repository URLs in the runtime

Follow these steps to add the repository URL:

> 1. Add new bundle repository URL:

Reviewer: ```suggestion
1. Add a new bundle repository URL:
```

 kubectl get pod -n kyma-system -l app=core-helm-broker
 ```

> Running Helm Broker triggers the Service Catalog synchronization automatically. New Service Classes should be after a half of minute.

Reviewer: ```suggestion
Running the Helm Broker triggers the Service Catalog synchronization automatically. New ServiceClasses appear after a half-minute.
```
